pkg/controller/secret/secret_controller.go

```diff
@@ -374,6 +374,18 @@ func createPagerdutyConfig(pagerdutyRoutingKey, clusterID string, clusterProxy s
 		detailsMap["cluster_id"] = ``
 		detailsMap["firing"] = ``
 		detailsMap["link"] = ``
+
+		return &alertmanager.PagerdutyConfig{
+			NotifierConfig: alertmanager.NotifierConfig{VSendResolved: true},
+			RoutingKey:     pagerdutyRoutingKey,
+			Severity:       `{{ if .CommonLabels.severity }}{{ .CommonLabels.severity | toLower }}{{ else }}critical{{ end }}`,
+			// Defaults to '{{ template "pagerduty.default.clientURL" . }}' when empty, however this contains
+			// the cluster name which is sensitive information for FedRAMP, so setting it to "ROSA"
+			ClientURL:   "ROSA",
```

Comment:
instead of duplicating the whole config, can we set a variable to the default, and then override it for FR?